src/datafactory/HISTORY.rst

```diff
@@ -7,6 +7,7 @@ Release History
 * Added Managed Identity (MI) authentication support for Azure Files
 * Introduced versioning property for better version control
 * Implemented Service Principal Certificate Authentication in RestService and SharePointOnlineList Linked 
+* Fix PostgreSqlV2 missing authenticationType property
```

Comment:
Do you want to release `1.0.3` after this PR merge? Are you waiting for any other features?